functions/Errors/Error.fmfn

```diff
@@ -14,10 +14,11 @@
  *		details = (object) JSON object including desired details
  *			at the time of error generation. This should help a developer
  *			determine why the error occured. See ErrorEnv() function.
+ *			It can also be a text string.
  *			(e.g., "creating a new invoice record")
  *
  * DEPENDENCIES:
- *		ErrorDescribeFmp()
+ *		ErrorMessage()
  *
  * NOTE:
```

Comment:
We should remove this note entirely if direct calls are being made to Error as opposed to calling ErrorApp or ErrorFmp which are simply convenience functions.

Reply:
Done.